packages/devtools_app/lib/src/timeline/flutter/flutter_frames_chart.dart

```diff
@@ -124,12 +172,6 @@ class FlutterFramesChartState extends State<FlutterFramesChart>
     _chartController.setViewPortOffsets(50, 10, 10, 30);
   }
 
-  @override
-  void dispose() {
-    _timer?.cancel();
-    super.dispose();
-  }
-
   void frameSelected(int frameIndex) {
     print('Bar Charted item selected frame index = $frameIndex');
   }
```

Comment:
can you hook up frame selection while you're here so that the selection is picked up by the listener in timeline_screen?

Reply:
Done.